README.md

```diff
@@ -17,47 +19,59 @@
 [![Scrutinizer Score](https://scrutinizer-ci.com/g/8p/EightPointsGuzzleBundle/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/8p/EightPointsGuzzleBundle/)
 [![License](https://poser.pugx.org/eightpoints/guzzle-bundle/license)](https://packagist.org/packages/eightpoints/guzzle-bundle)
 
-This bundle integrates [Guzzle 6.x][1] into Symfony. Guzzle is a PHP framework for building RESTful web service clients.
+This bundle integrates [Guzzle 6.x][1] into Symfony[16]. Guzzle is a PHP library for building RESTful web service clients.
 
 GuzzleBundle follows semantic versioning. Read more on [semver.org][2].
 
 ----
 
-## Requirements
- - PHP 7.1 or above
- - Symfony 4.0 or above
- - [Guzzle PHP Framework][1] (included by composer)
+## Prerequisites
+ - PHP 7.1 or higher
+ - Symfony 4.0 or 5.0
```

Comment:
With this definition is not clear, does it support SF 4.1, for example. My suggestion is to use `Symfony 4 or 5` or `Symfony 4.x or 5.x` or `Symfony v4/v5`

Reply:
Agreed. Changed to `Symfony 4.x or 5.x`.